src/state/walletConnectRequests/index.ts

```diff
@@ -0,0 +1,68 @@
+import { reverse, sortBy, values } from 'lodash';
+import { WalletconnectRequestData } from '@/walletConnect/types';
+import { createStore } from '../internal/createStore';
+import { omitFlatten } from '@/helpers/utilities';
+
+interface RequestsState {
+  [requestId: number]: WalletconnectRequestData;
+}
+
+export interface WalletConnectRequestsState {
```

Comment:
new store that replaces `redux/requests` and exposes the same methods for adding a request, removing a request, and getting a sorted list of the WC requests to display